airesources/Kotlin/src/kotlin/halite/Collision.kt

```diff
@@ -0,0 +1,55 @@
+package halite
+
+object Collision {
+    /**
+     * Test whether a given line segment intersects a circular area.
+     *
+     * @param start  The start of the segment.
+     * @param end    The end of the segment.
+     * @param circle: The circle to test against.
```

Comment:
I think you are a bit out of date from latest Java update history - the ':' was cleaned up recently. Can you go through https://github.com/HaliteChallenge/Halite-II/commits/master/airesources/Java to make sure you are caught up with recent changes?

After that I'll keep Kotlin in mind and update it as well if/when touching Java later.

Reply:
Sure, I can clean this up. But, going forward, wow in sync do we actually want to keep the two starter kits?

Reply:
If it's obviously same code/text then I'll keep it in sync. I.e. if there is like a math bug in Java I'll check all of the kits derived from it and fix it there if bug is still there. But naturally if design of the kit starts to diverge because of the language - that's actually great.